test/testinput/bufr_ncep_prepbufr_adpupa.yaml

```diff
@@ -0,0 +1,425 @@
+            group_by: prepbufrDataLvlCat
+
+          airTemperature:
+            query: "*/PRSLEVEL/T___INFO/T__EVENT/TOB"
+            group_by: prepbufrDataLvlCat
+            transforms:
+              - offset: 273.15
+          airTemperatureQualityMarker:
+            query: "*/PRSLEVEL/T___INFO/T__EVENT/TQM"
+            group_by: prepbufrDataLvlCat
+
+          heightOfObservation:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT/ZOB"
+            group_by: prepbufrDataLvlCat
+          heightQualityMarker:
+            query: "*/PRSLEVEL/Z___INFO/Z__EVENT/ZQM"
+            group_by: prepbufrDataLvlCat
+
+          windEastward:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/UOB"
+            group_by: prepbufrDataLvlCat
+          windNorthward:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/VOB"
+            group_by: prepbufrDataLvlCat
+          windQualityMarker:
+            query: "*/PRSLEVEL/W___INFO/W__EVENT/WQM"
+            group_by: prepbufrDataLvlCat
+
+          pressureError:
+            query: "*/PRSLEVEL/P___INFO/P__BACKG/POE"
+            group_by: prepbufrDataLvlCat
+            transforms:
+              - scale: 100
+          relativeHumidityError:
+            query: "*/PRSLEVEL/Q___INFO/Q__BACKG/QOE"
+            group_by: prepbufrDataLvlCat
+          airTemperatureError:
+            query: "*/PRSLEVEL/T___INFO/T__BACKG/TOE"
+            group_by: prepbufrDataLvlCat
+            transforms:
+              - offset: 273.15
+          windError:
+            query: "*/PRSLEVEL/W___INFO/W__BACKG/WOE"
+            group_by: prepbufrDataLvlCat
+
+          lonProfileLevel:
+            query: "*/PRSLEVEL/DRFTINFO/XDR"
+            group_by: prepbufrDataLvlCat
+          latProfileLevel:
+            query: "*/PRSLEVEL/DRFTINFO/YDR"
+            group_by: prepbufrDataLvlCat
+          timeCycleProfileLevel:
+            query: "*/PRSLEVEL/DRFTINFO/HRDR"
+            group_by: prepbufrDataLvlCat
+
+          seaSurfaceTemperature:
+            query: "*/SST_INFO/SSTEVENT/SST1"
+          seaSurfaceTemperatureQualityMarker:
+            query: "*/SST_INFO/SSTEVENT/SSTQM"
+
+          seaSurfaceTemperatureError:
+            query: "*/SST_INFO/SSTBACKG/SSTOE"
+
+          depthBelowSeaSurface:
+            query: "*/SST_INFO/DBSS_SEQ/DBSS"
+
+
+    ioda:
+      backend: netcdf 
+      obsdataout: "./testrun/bufr_ncep_prepbufr_adpupa.nc"
+
+
+#      dimensions:
+##       - name: prslevel_Dim -> Level
+#        - name: Level
+#          path: "*/PRSLEVEL"
+#        - name: cloudseq_Dim
+#          path: "*/CLOUDSEQ"
+#        - name: pevent_Dim
+#          path: "*/PRSLEVEL/P___INFO/P__EVENT"
+#        - name: qevent_Dim
+#          path: "*/PRSLEVEL/Q___INFO/Q__EVENT"
+#        - name: tevent_Dim
+#          path: "*/PRSLEVEL/T___INFO/T__EVENT"
+#        - name: zevent_Dim
+#          path: "*/PRSLEVEL/Z___INFO/Z__EVENT"
+#        - name: wevent_Dim
+#          path: "*/PRSLEVEL/W___INFO/W__EVENT"
+#
```

Comment:
We can use these dimensions.  @rhoneyager will add this to ObsSpace.yaml when we send this yaml for PR (to be merged to develop) in the future.